### PR TITLE
[Fixed] Responsiveness of Latest Community News Section

### PR DESCRIPTION
**Pull request by itsmanojthapa**
Solved #205 

Before: 
image

After:
image

<a href="https://gitpod.io/#https://github.com/RocketChat/RC4Community/pull/209"></a>

### CODE REVIEW

**Comment by Palanikannan1437** [comment]
Hey @itsmanojthapa, awesome work there...thanks a ton for keeping the design and making it responsive at the same time! 

Could you please fix the mismatched heights of the items while you're at it? Thank you 😄

image

**Comment by itsmanojthapa** [comment]
Is this fine 


pic 1:
image

pic: 2:
image

pic 3:
image


pic 4: 
image

**Comment by Palanikannan1437** [comment]
LGTM @itsmanojthapa!!, thanks for your PR!

@Sing-Li it's ready to be merged, thank you!

File: app/components/newscarousel.js [hunk 1]
```diff
@@ -58,6 +58,7 @@ const Item = (props) => {
           height={140}
           src={props.item.attributes.imageUrl}
           alt={props.item.attributes.name}
+          className={`${styles.imgItm}`}
```
Comment (Dnouv) [inline app/components/newscarousel.js:61]: Thanks for the PR. Just a minor suggestion. In `className`, you can also define it as `className={styles.imgItm}`.

Comment (itsmanojthapa) [reply app/components/newscarousel.js]: Thank you for your suggestion @Dnouv

Comment (Dnouv) [reply app/components/newscarousel.js]: LGTM! Thanks for the quick changes.

**Pull request merged**